lib/remote/zone.ti

```diff
@@ -9,7 +9,7 @@ namespace icinga
 
 class Zone : ConfigObject
 {
-	[config, navigation] name(Zone) parent (ParentRaw) {
+	[config, no_user_modify, navigation] name(Zone) parent (ParentRaw) {
```

Comment:
This attributes don't affect the object name, so from that point of view, changing it is fine.

Please try to figure out whether it's safe to change this at runtime, keep or revert this change accordingly and give the reasoning for it.